spec/mongoid/enum_spec.rb

```diff
@@ -174,4 +192,21 @@ class User
       end
     end
   end
+
+  describe ".configuration" do
+    it "returns Configuration object" do
+      expect(Mongoid::Enum.configuration)
+          .to be_instance_of Mongoid::Enum::Configuration
+    end
+    it "returns same object when called multiple times" do
+      expect(Mongoid::Enum.configuration).to be Mongoid::Enum.configuration
+    end
+  end
+
+  describe ".configure" do
+    it "yields configuration if block is given" do
+      expect { |b| Mongoid::Enum.configure &b }
+          .to yield_with_args Mongoid::Enum.configuration
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 4) spaces for indenting an expression spanning multiple lines.